hs-dear-imgui.cabal

```diff
@@ -1,5 +1,5 @@
 cabal-version: 3.0
-name: dear-imgui
+name: hs-dear-imgui
```

Comment:
I'd rather keep this library named `dear-imgui`, it's implicit that it's Haskell.

Reply:
my fault, there was some confusion between haskell.nix, cabal, myself, and what "name"s were being used where. fixed now.

Reply:
actually not.  It seems like haskell.nix expects some sort of parity between package names, cabal file names, and haskell.nix "name" identifiers.  after dropping `hs-` `nix-shell` fails with: 
```
error: The option `packages.dear-imgui.package.identifier.version' is used but not defined.
```
Do you know where the mixup might be?